### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by darxriggs**
I suggest to review it commit by commit.

### CODE REVIEW

File: src/main/java/com/nirima/jenkins/plugins/docker/DockerContainerWatchdog.java [hunk 1]
```diff
@@ -329,7 +329,7 @@ private void cleanUpSuperfluousContainers(DockerClient client, Map<String, Node>
     }
 
     private boolean isStillTooYoung(Long created, Instant snapshotInstant) {
-        final Instant createdInstant = Instant.ofEpochSecond(created.longValue());
+        final Instant createdInstant = Instant.ofEpochSecond(created);
```
[the remark this answers is not in the document]
Comment (darxriggs) [reply src/main/java/com/nirima/jenkins/plugins/docker/DockerContainerWatchdog.java]: Thanks for your thoughts. If you fear debugging issues I probably better revert this change than to add explanations in the form of comments.

**Comment by darxriggs** [comment]
I updated the unboxing commit and added another one for fixing some Javadocs.

**Comment by darxriggs** [comment]
There seems to be a problem with the Jenkins CI infrastructure.

**Comment by pjdarton** [comment]
I agree; those failures don't look like they've got anything to do with your code.
I've encountered this kind of problem before when working on my own PRs, and the advice that I was given by other Jenkins folks was that you prod the jenkinsci servers into having another attempt at building it by doing the following:
1. Close this PR
1. Wait for 10 minutes (this is important - you need to wait at least 10 minutes).
   During this time, jenkinsci should see that the PR is closed and forget about it.
1. Re-open the PR
1. Wait for up to 10 minutes
   During this time, jenkinsci should see that there is a PR that needs to be built, and try to build it.

If that doesn't resolve things then it's a good idea to (politely) let people know on the [jenkins-infra](http://webchat.freenode.net/?channels=jenkins-infra) channel.

**Comment by darxriggs** [comment]
Hi @pjdarton, is there anything else I should do? Your requested changes have already been incorporated.

**Comment by pjdarton** [comment]
Sorry for the delay; real life issues (moving house) + backlog of work at my day-job = delays responding here.

**Comment by pjdarton** [comment]
Re: https://github.com/jenkinsci/docker-plugin/pull/692/commits/40d223979e0217bb6b11d2a158015871646e4681
OK, I'm happy with that.

Re: https://github.com/jenkinsci/docker-plugin/pull/692/commits/716752a3665e87d7b5279cbe26e5786b00a7160e
These seem to be the same changes as reviewed earlier; they're still good.

Re: https://github.com/jenkinsci/docker-plugin/pull/692/commits/08e090d107ddbbe2157c9aa0930f1c2f6471367a
Well spotted; those changes look good to me.

**Pull request merged**